test: integrate runtime-tests as a helper library (#549)
paritytech/substrate-api-sidecar: runtime-tests/README.md

## Summary

> This is a helper library for Sidecar to run runtime tests against specific chains, at certain blocks. 

Reviewer: What does "runtime tests" mean here?

## Testing

> Below I will go over testing for Polkadot. These instructions are specific to runtime-tests against one chain. 

Reviewer: ```suggestion
The below instructions are specific to runtime-tests against one chain. 
```

If you are looking to run the runtime-tests against all 3 chains (Polkadot, Kusama, Westend) then run `yarn test:init-runtime-tests` in 
the root directory of sidecar.

### Polkadot 

Lets first get sidecar ready in a seperate terminal.

```
$ cd substrate-api-sidecar
$ git checkout <your_branch>

> $ export SAS_SUBSTRATE_WS_URL=wss://rpc.polkadot.io

Reviewer: ```suggestion
$ export SAS_SUBSTRATE_WS_URL=<network archive node>
```

$ yarn

> $ yarn dev

Reviewer: ```suggestion
$ yarn build && yarn start
```

```

> Sidecar should be now connected to the node, and running successfully. If you are having issues running sidecar and are receiving some odd behavior, contact the maintainers, or file an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues).

Reviewer: ```suggestion
Sidecar should now connected to the node and running successfully. If you are having issues running sidecar and are receiving some odd behavior, contact the maintainers, or file an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues).
```

> Sidecar should be now connected to the node, and running successfully. If you are having issues running sidecar and are receiving some odd behavior, contact the maintainers, or file an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues).

Reviewer: ```suggestion
Sidecar should be now connected to the node, and running successfully. If you find a bug file an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues).
```

> Now lets run our runtime tests against polkadot. Go to the seperate terminal and run:

Reviewer: ```suggestion
Now lets run our runtime tests against polkadot. Go to a separate terminal and run:
```
?
